wheels/cloudauthz/meta.yml

```diff
@@ -0,0 +1,8 @@
+---
+
+type: wheel
+name: cloudauthz
+purepy: true
+version: 0.1.0
+src:
+    - https://files.pythonhosted.org/packages/ac/44/20d0e6865db1fafc6d3f0fd2ee767d42e287914be5e4d20649a22d1494b6/cloudauthz-0.1.0.tar.gz
```

Comment:
Also not needed as long as `pip download <package>` is all you need to fetch the package source (so anything in PyPI).

Reply:
@natefoo Even better!